ROSCO/src/Functions.f90

```diff
@@ -67,92 +67,87 @@ REAL(DbKi) FUNCTION ratelimit(inputSignal, inputSignalPrev, minRate, maxRate, DT
     END FUNCTION ratelimit
 
 !-------------------------------------------------------------------------------------------------------------------------------
-    REAL(DbKi) FUNCTION PIController(error, kp, ki, minValue, maxValue, DT, I0, reset, inst)
+    REAL FUNCTION PIController(error, kp, ki, minValue, maxValue, DT, I0, piP, reset, inst)
```

Comment:
Same issue here. I don't think it's causing the instability in the tests, but maybe.